Clear command logging

CalebABG opened the pull request:
This addresses issue #188 for adding proper logging to the clear command when a logging channel is set

CalebABG committed:
This addresses issue #188 for adding proper logging to the clear command when a logging channel is set

CalebABG committed:
Updated title and footer + added back original embed for responding

VelvetToroyashi commented on src/Silk.Core.Discord/Commands/General/ClearCommand.cs:
```diff
@@ -1,35 +1,76 @@
+            catch (NotFoundException) {}
+        }
+
+        private async Task<GuildConfig> GetOrCreateGuildConfig(CommandContext ctx)
```
This entire thing could just be replaced with GetOrCreate, as it handles similar internal logic.

VelvetToroyashi commented on src/Silk.Core.Discord/Commands/General/ClearCommand.cs:
```diff
@@ -1,35 +1,76 @@
+            GuildConfig guildConfig = await GetOrCreateGuildConfig(ctx);
+            
+            ulong loggingChannelId = guildConfig.LoggingChannel;
+            DiscordChannel? loggingChannel = ctx.Guild.GetChannel(loggingChannelId) ?? ctx.Channel;
```
~~use null coalescing operator (`?.`) instead, and~~ log the original embed (x messages cleared) to the channel regardless of logging is setup.

CalebABG committed:
Slight refactor

- Made use of GetOrCreateGuildRequest (needed to call '.Include(g => g.Configuration)' to include GuildConfig)
- Extracted methods for Embeds

CalebABG committed:
Merge branch 'development' of https://github.com/VelvetThePanda/Silk into clearCommandLogging
